### DISCUSSION TITLE
我大概一辈子都不会忘记你吧

### DISCUSSION THREAD

**Discussion by YDwang1459** [share121/inter-knot#3234] [其他]
![image_download_1724008607052](https://github.com/user-attachments/assets/62ab6246-b84d-4f24-933b-32089aacf814)

**Comment by YDwang1459** [comment]
![image_download_1724008605243](https://github.com/user-attachments/assets/a99167be-8c6b-4ac8-9277-21645547dc7b)

**Comment by Nie7bai** [comment]
牢卡，不——
